index.html

```diff
@@ -20,377 +20,274 @@
-           <span>Html</span>
-          </div>
-          <div class="third-tag2">
-           <span>bootstrap</span>
-          </div>
-          <div class="third-tag3">
-           <span>Ruby</span>
-          </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
-        </div>
-
-
-
-
-        
-
-      </section>
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="fourth-tags">
-          <div class="fourth-tag">
-           <span>Html</span>
-          </div>
-          <div class="fourth-tag2">
-           <span>bootstrap</span>
+      <div class="card-works">
+        <div class="feature">
+          <div class="img-placeholder">
+            <img src="assets/img-placeholder.png" alt=""/>
           </div>
-          <div class="fourth-tag3">
-           <span>Ruby</span>
-          </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
-        </div>
-
-
-
-
-        
-
-      </section>
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="fifth-tags">
-          <div class="fifth-tag">
-           <span>Html</span>
-          </div>
-          <div class="fifth-tag2">
-           <span>bootstrap</span>
-          </div>
-          <div class="fifth-tag3">
-           <span>Ruby</span>
+          <div class="right-block">
+            <h2>Multi-Post Stories</h2>
+            <p>A daily selection of privately personalized reads; 
+              no accounts or sign-ups required. 
+              has been the industry's standard dummy text ever since the 1500s, 
+              when an unknown printer took a standard dummy text.
+            </p>
+            <div class="tag">
+              <ul>
+                <li>Css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </div>
+            <div class="block-btn">
+              <a href="#">See Project</a>
+            </div>
           </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
         </div>
 
-
-
-
-        
-
-      </section>
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="sixth-tags">
-          <div class="sixth-tag">
-           <span>Html</span>
+        <div class="card-1">
```

Comment:
- [Optional ] I can see that your style is not applied to your project you can check well your style link and this was addressed in the previous review, kindly have a look at this :point_down: 
![desktopv](https://user-images.githubusercontent.com/88894525/206801099-101c07b9-147d-4866-8a23-fae107a32c16.png)